packages/playwright/src/index.ts

```diff
@@ -102,23 +106,23 @@ const playwrightFixtures: Fixtures<TestFixtures, WorkerFixtures> = ({
     if (connectOptions) {
       const browser = await playwright[browserName].connect({
         ...connectOptions,
-        exposeNetwork: connectOptions.exposeNetwork ?? (connectOptions as any)._exposeNetwork,
+        exposeNetwork: connectOptions.exposeNetwork ?? (connectOptions as ConnectOptionsInternal)._exposeNetwork,
```

Comment:
This is a wrong cast. `_exposeNetwork` is here to support old api, not to conform with `ConnectOptionsInternal`. That's just a coincidence.